datahub-web-react/src/app/domain/DomainsList.tsx

```diff
@@ -85,6 +103,77 @@ export const DomainsList = () => {
         }, 2000);
     };
 
+    const logoIcon = entityRegistry.getIcon(EntityType.Domain, 12, IconStyleType.ACCENT);
+    const allColumns = [
+        {
+            title: 'Name',
+            dataIndex: '',
+            key: 'name',
+            sorter: (sourceA, sourceB) => {
+                return sourceA.name.localeCompare(sourceB.name);
+            },
+            render(record: DomainEntry) {
+                return (
+                    <Link to={record.url}>
+                        {logoIcon}
+                        <DomainNameContainer>
+                            <Typography.Text>{record.name}</Typography.Text>
+                        </DomainNameContainer>
+                        <Tooltip title={`There are ${record.entities} entities in this domain.`}>
+                            <Tag>{record.entities} entities</Tag>
+                        </Tooltip>
+                    </Link>
+                );
+            },
+        },
+        {
+            title: 'Owners',
+            dataIndex: '',
+            width: '10%',
+            key: 'owners',
+            render(record: DomainEntry) {
+                return (
+                    <>
+                        <DomainOwners ownership={record.owners} />
+                    </>
+                );
+            },
+        },
+        {
+            title: '',
+            dataIndex: '',
+            width: '5%',
+            align: 'right' as AlignType,
+            key: 'menu',
+            render(record: DomainEntry) {
+                return (
+                    <>
```

Comment:
Let's try to extract each column type into a separate file.

We can follow the example of IngestionSourceListColumns.tsx